controllers/start.go

```diff
@@ -51,6 +53,10 @@ func StartManager(cfg ControllerConfig) error {
 		logger.Error(err, "failed to register ACR")
 		return err
 	}
+	if err := gcr.Register(context.TODO(), logger, registry); err != nil {
```

Comment:
Since we don't have a `context` to really use, should it just be an implementation detail of Register for now?

Reply:
Normally there would be a context when starting up usually tied to shutdown.
Rather than keep it buried, we have a marker for future improvement.
Too much of this code is self contained especially the config, startup bits. But that isn't for this PR.